src/api/index.tsx

```diff
@@ -266,3 +276,50 @@ export const getDiff = async ({
     query: path ? { file: path } : undefined,
   });
 };
+
+export const createOrUpdateComment = async ({
+  /* istanbul ignore next */
+  _callApi = callApi,
+  addonId,
+  apiState,
+  cannedResponseId,
+  comment,
+  commentId,
+  fileName,
+  line,
+  versionId,
+}: {
+  _callApi?: typeof callApi;
```

Comment:
I prefer to see the types inline because it makes the function easier to read (IMO). Everything is in the same place. Historically, we have only been making separate type definitions if we *have* to but there are probably some exceptions to that. I have always started with inline types until I could no longer do so (for sharing purposes, exports, etc).